### DISCUSSION TITLE
Its not working for me

### DISCUSSION THREAD

**Discussion by GSVil** [itsToggle/plex_debrid#41] [Q&A] [answered]
Hope someone can point me to what I'm missing....

1) So I have rclone running....terminal output below:
C:\>rclone cmount your-remote: X: --dir-cache-time=10s --vfs-cache-mode=full
The service rclone has been started.

2) I have the plex_rd.py setup and running...terminal output below:
*1) Run
*2) Settings
*3) Ignored Media
*4) Scraper

*Type exit to quit.

*Choose an action: 1

But nothing is happening in PLEX, I add something to the "watchlist" and nothing happens in PLEX or anywhere else.

I'm really confused as to what I'm not doing, or what I missed...

Thanks in advance for any help given...

Kind Regards
Geo

**Comment by itsToggle** [comment, marked as answer]
Hey, did you press enter after you typed '1' ? :)

Edit: this was solved by a few updates to plex_debrid.

**Reply by GSVil** [reply]
Yes...I just see a flashing prompt. :-)

**Reply by GSVil** [reply]
I've tried option 4 Scraper, and that works fine with my Real-Debrid account....so I know the API and scraping works.

**Reply by itsToggle** [reply]
hm.. does it say anything at all when you press '1'? You should see the plex_debrid logo and a log like this:


               __                  __     __         _     __
        ____  / /__  _  __    ____/ /__  / /_  _____(_)___/ /
       / __ \/ / _ \| |/_/   / __  / _ \/ __ \/ ___/ / __  /
      / /_/ / /  __/>  <    / /_/ /  __/ /_/ / /  / / /_/ /
     / .___/_/\___/_/|_|____\__,_/\___/_.___/_/  /_/\__,_/
    /_/               /_____/
    
    
    
    Type 'exit' to return to the main menu.
    [2022-07-18 15:17:38.768830] getting entire plex library ... done
    [2022-07-18 15:17:39.702164] getting all plex watchlists ... done

**Reply by GSVil** [reply]
No, it only shows it after I type Exit....

![image](https://user-images.githubusercontent.com/69642887/179522308-290e3529-23c3-4ba9-bcd4-e3aafff6aa00.png)

**Reply by GSVil** [reply]
I don't have overseer installed...so not sure if that is why its giving me an error...

**Reply by itsToggle** [reply]
I see! I'm sorry about this error, someone else reported it a few minutes ago aswell and I've updated the script with a fix.
Could you download the newest version of the script and try again? If you leave the settings.json file intact, you dont need to run the initial setup again :)

**Reply by itsToggle** [reply]
I'm not sure why the script isn't printing anything while it's running, what kind of terminal are you using?

**Reply by GSVil** [reply]
OK I'll try the updated script....
Its the GUI version of the terminal that comes with the Python install on windows....I'll try the CMD-Prompt version and see what it does.

**Reply by itsToggle** [reply]
alright i might have an idea why its not printing. Ill update the script again.

**Reply by GSVil** [reply]
That sorted it...thanks:
![image](https://user-images.githubusercontent.com/69642887/179525637-50124cd6-40a0-412b-bade-735b9aedac05.png)

**Reply by GSVil** [reply]
Just by the way, I had to install quite a few of the libraries with pip3 to get the script working after a fresh python install. Also your script is only Python3 compatible...I tried running it on version 2, which I already had installed and it failed to validate the syntax. Maybe two items you might want to detail in your install procedure.

**Reply by itsToggle** [reply]
noted, thanks! I've been trying forever to figure out how to create a requirements.txt for pip, it always returned errors when I tried. Perhaps I'll give it a shot again.
I just noticed that in your screenshot the '&' isn't properly replaced by 'and', which is why there aren't any releases found - I'll release a quick fix for that aswell.

**Reply by GSVil** [reply]
Cool, will you let me know when you have made the change... Thanks for the help getting this to work.

**Reply by itsToggle** [reply]
alright, the '&' should now be propely renamed. Before you run the script again, choose option 3) Ignored media and remove the ingored epsiodes from 'superman & lois' :)

**Reply by GSVil** [reply]
cool...will try it

**Reply by GSVil** [reply]
OK working perfectly now... Again thanks a lot. All i have to do now is schedule everything to start up automatically. Really nice work, thanks again for the code and the quick response in fixing it.

**Reply by GSVil** [reply]
Hi There, I'm back.... I need some help again :-)

It appears when you use "rclone" it creates some folders under your user-name. The problem is that its on my C: drive which is really small, which means I'm running out of space really quickly. This is the statement I used:
rclone cmount your-remote: X: --dir-cache-time=10s --vfs-cache-mode=full
Is there any way to tell it where to create your-remote other than on my C: drive?

**Reply by itsToggle** [reply]
Yeah rclone creates a 'cache' directory, where files that you open are temporarily cached. You can define the location of this cache drive, I think the mounting tag is '--vfs-cache-dir your-location' - but I would refer you to the official rclone 'mount' documentation for this.

You can also define a maximum size for the cache drive, I do this on my vps because I have very limited space aswell. The mounting tag is '--vfs-max-cache-size 10G' for a 10Gb maximum.

Hope this helps :)

**Reply by GSVil** [reply]
Cool...will try it.
Then 2 more things...please:

1) On the scraping, I see it sometimes scrapes/retrieves movies/series that are similar to the title you add to your Watchlist . For example I was looking for the NCIS, and it scraped NCIS LA,  NCIS New Orleans and some other stuff that had NCIS in the name as well. Is that a limitation? And if it is, how do I get rid/delete the ones I don't want? 

2) Then this afternoon I was trying to add a really old series for my wife, and the parsing fails.....see attachment.
![image](https://user-images.githubusercontent.com/69642887/180245851-e81d41d0-cf19-4be3-9f93-16879525590a.png)

**Reply by itsToggle** [reply]
I would generally recommend you install Jackett as your scraper and remove the "legacy" scrapers for 1337x and rarbg :) you can add a lot more torrent indexers with Jackett and its overall faster than the scrapers I've written.

1.) That sadly is a limitation. I've not yet implemented a scraping method by IMDB (or similar) ID, so the scraper is searching for releases by title only. Releases that match the title closely are accepted aswell, since sometimes that's the only way to find releases. An example for this is the following: the scraper will search for "some.show.s01.", but the releases are named "some.show.2020.season.1.s01" - so to accept these releases, some title deviation is permitted. Releases with the least title deviation are prefered, but sometimes a wrong release can be downloaded. 
I will work on this issue :)

2.) sometimes there just aren't any releases for old series or movies. What you can do is either install Jackett as your scraper to search more torrent indexers, or manually search for this show :) the legacy scraper for rarbg does produce the parse error from time to time, nothing to worry about.

**Reply by GSVil** [reply]
:-)  "you can add a lot more torrent indexers with Jackett" .... I've installed Jasckett, but I dont know what to do with it.. LOL!! I'll just stick with the legacy scrapers you gave me...that's unless you have a how-to that I can follow :-)  Everything appears to function properly now, but I have some more questions if you have time for me?

1) My firewall is reporting huge internet use since I've installed your stuff.... I haven't really started watching anything in PLEX, I've just added a few shows and movies....but every day my firewall is reporting 50 to 80GB internet use. Why is this happening, and how do i stop it.
2) How do i remove the titles/files that were incorrectly scraped.....just deleting them form my X-Drive doesn't work.....

Sorry if these questions are stupid, but I don't really know what I'm doing. I'm a fist time user of Real-Debrid ,which was recommended by  a friend that uses it with Kodi. I only stumbled on your project because I'm using PLEX and not Kodi so I was looking for a way to use Real-Debrid in Plex.

**Reply by itsToggle** [reply]
The official Jackett documentation should help you out with setting it up. You should just be able to add any indexer you want to jackett (rarbg, 1337x, pirate bay, etc).

Plex does some metadata analyses in the background. You can turn off most of the media analyses as suggested in the readme, but as long as rclone is mounted there will always be some traffic.

Have you mounted rclone with the official rclone software or with my fork? As stated in the readme, deleting does not work with the official rclone software. You can either switch to my fork for mounting or delete the torrents manually on the realdebrid webpage. You can also try a different software for mounting the realdebrid WebDAV.
